groups/hsp-iit.yml

```
@@ -30,6 +30,7 @@ hsp-iit/purchases:
   - "drdanz"
   - "GiuliaP"
   - "fbottarel"
+  - "xEnVrE"
```

```suggestion
  - "Arya07"
```

Let's shortcut. We can always fix it later if required.

---

It was for @Arya07 correct.